packages/enzyme-adapter-react-16.1/src/ReactSixteenOneAdapter.js

```diff
@@ -262,6 +263,19 @@ class ReactSixteenOneAdapter extends EnzymeAdapter {
       getNode() {
         return instance ? toTree(instance._reactInternalFiber).rendered : null;
       },
+      simulateError(nodeHierarchy, rootNode, error) {
+        const { instance: catchingInstance } = nodeHierarchy
+          .find(x => x.instance && x.instance.componentDidCatch) || {};
```

Comment:
If I have multiple error boundary components in my tree, will this throw on the closest one to the current node?

Reply:
It should throw on the first one it finds as it traverses upwards.